.tool-versions

```diff
@@ -4,3 +4,4 @@ python 3.11.4
 terraform 1.5.7
 jq 1.7
 yq 4.34.2
+nodejs 20.9.0
```

Comment:
Is this now needed on the packer image? Starting to sound like that ticket could do with being brought in soon haha

Reply:
ah yeah i'll add that to the ticket